mono/utils/mono-error.h

```diff
@@ -50,27 +50,36 @@ enum {
 	MONO_ERROR_CLEANUP_CALLED_SENTINEL = 0xffff
 };
 
+typedef union _MonoErrorExternal MonoErrorExternal;
+typedef union _MonoErrorInternal MonoErrorInternal;
+
 #ifdef _MSC_VER
 __pragma(warning (push))
 __pragma(warning (disable:4201))
 #endif
 
 /*Keep in sync with MonoErrorInternal*/
-typedef union _MonoError {
+union _MonoErrorExternal {
 	// Merge two uint16 into one uint32 so it can be initialized
 	// with one instruction instead of two.
 	uint32_t init;
 	struct {
 		uint16_t error_code;
-		uint16_t private_flags; /*DON'T TOUCH */
+		uint16_t hidden_0; /*DON'T TOUCH */
```

Comment:
This is how it used to be. I changed it. I don't remember why. I'll check git blame.

Reply:
No reason obvious in the PR. Maybe to see better in debugger?